optuna/multi_objective/samplers/_motpe.py

```diff
@@ -37,16 +37,12 @@ class MOTPEMultiObjectiveSampler(BaseMultiObjectiveSampler):
         consider_prior:
             Enhance the stability of Parzen estimator by imposing a Gaussian prior when
             :obj:`True`. The prior is only effective if the sampling distribution is
-            either :class:`~optuna.distributions.UniformDistribution`,
-            :class:`~optuna.distributions.DiscreteUniformDistribution`,
-            :class:`~optuna.distributions.LogUniformDistribution`,
+            either :class:`~optuna.distributions.FloatDistribution`,
```

Comment:
In my understanding, `UniformDistribution`, `DiscreteUniformDistribution` and `LogUniformDistribution` are also effective. But I also think it is reasonable to me that only show `FloatDistribution` (not showing deprecated distributions). I'd like other developer's opinion.

https://github.com/optuna/optuna/pull/3275/files#diff-01b5f9e9ffd93b1e908f80f16fae814f065d060e908a779670e905dc30109007R232

Reply:
I think it would be better to just mention FloatDistribution as in the current change; it would help users who have checked the document to reduce the chances of using the old distribution as much as possible.

Reply:
Kept the changes from #3181